ros2_batch_job/__main__.py

```diff
@@ -413,11 +438,12 @@ def run(args, build_function, blacklisted_package_names=None):
 
     job = None
 
-    args.white_space_in = args.white_space_in or []
     args.workspace = 'work space' if 'workspace' in args.white_space_in else 'ws'
-    args.sourcespace = 'source space' if 'sourcespace' in args.white_space_in else 'src'
-    args.buildspace = 'build space' if 'buildspace' in args.white_space_in else 'build'
-    args.installspace = 'install space' if 'installspace' in args.white_space_in else 'install'
+    # Add white space to path. Logic above should prevent conflict of manually specified path and '--white-space-in'
```

Comment:
I think we can remove all of this logic now, right?

Reply:
Correct, I removed stuff